scipy/stats/tests/test_distributions.py

```diff
@@ -2969,6 +2969,77 @@ def test_entropy_transposed(self):
                                   stats.entropy(pk, qk, axis=1))
 
 
+class TestDifferentialEntropy(object):
+    """
+    Results are compared with the R package vsgoftest.
+
+    """
+
```

Comment:
Great way to test, and thanks for including the package name! Could you include example code as a comment so that these are very easy to verify?
```suggestion
    Results are compared with the R package vsgoftest.

    """
    # library(vsgoftest)
    # options(digits=16)
    # samp <- c(1, 2, 3, 4, 5)
    # entropy.estimate(x = samp, window = 1)
```
Is there a way to check with a different base?

Reply:
I have tested it against the `entropy.estimate` function, but I had to copy the generated samples, of size 100, so I do not think that it is very sensible to put them as a comment.

It seems that `entropy.estimate` only uses the natural base, so I would not be able to test this parameter against it.

Reply:
> I do not think that it is very sensible to put them as a comment.

Agreed. I prefer to be sensible, too. That's why the code suggestion has just five integers : )

If you agree that it would be useful to future reviewers, all that's needed is to click "Commit suggestion".

![image](https://user-images.githubusercontent.com/6570539/110227593-c9cd0d00-7eae-11eb-9c76-1d4e90007337.png)

Reply:
I extended the comment with the code that I used in R.